mapper/cumulocity/c8y_smartrest/src/smartrest_serializer.rs

```diff
@@ -96,13 +118,23 @@ impl<'a> SmartRestSerializer<'a> for SmartRestSetOperationToExecuting {}
 pub struct SmartRestSetOperationToSuccessful {
     pub message_id: &'static str,
     pub operation: &'static str,
+    pub url: Option<String>,
 }
 
 impl SmartRestSetOperationToSuccessful {
     fn new(operation: CumulocitySupportedOperations) -> Self {
         Self {
             message_id: "503",
             operation: operation.into(),
+            url: None,
+        }
+    }
+
+    pub fn new_with_file(operation: CumulocitySupportedOperations, url: &str) -> Self {
+        Self {
+            message_id: "503",
+            operation: operation.into(),
+            url: Some(url.into()),
```

Comment:
As far as I understand this smartrestconstruct this `url` is actually more generic parameter which can be anything for a parametrised response.

Let's make it more generic by just renaming the components:
```suggestion
    pub fn new_with_parameter(operation: CumulocitySupportedOperations, response_parameter: &str) -> Self {
        Self {
            message_id: "503",
            operation: operation.into(),
            response_parameter: Some(url.into()),
```